core/block/import/source/source.go

```diff
@@ -16,6 +16,7 @@ var extensions = []string{".md", ".csv", ".txt", ".pb", ".json", ".html"}
 
 type Source interface {
 	GetFileReaders(importPath string, ext []string, includeFiles []string) (map[string]io.ReadCloser, error)
+	Close()
```

Comment:
We use method Close() to close all imported files and all files from source archive or directory